docs/articles/documentation/reference/testcafe-api/runner/compileroptions.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title: Runner.compilerOptions Method
+permalink: /documentation/reference/testcafe-api/runner/compileroptions.html
+---
+# Runner.compilerOptions Method
+
+Customizes the settings of the [TypeScript compiler](../../../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).
```

Comment:
Use the same phrase for all interfaces. Choose the single variant from 
* `Modifies TypeScript compilation settings`, 
* `Configures the [TypeScript compiler](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options)` 
* `Customizes the settings of the [TypeScript compiler]`.